src/components/Modal/GPOSModal/DepositWithdraw/DepositWithdraw.jsx

```diff
@@ -205,11 +205,9 @@ class DepositWithdraw extends PureComponent {
       // Power Down
       if (this.props.action === 1.2) {
         // Check if current vested balance can cover the fee
-        if (totalGpos < (fees.down)) {
```

Comment:
If available balance is 0. It allows the user to type in 999999999 in the GPOS power up input. It should not be allowing the user to type in a number greater than 0, and displaying an error msg.
![Screen Shot 2020-01-06 at 3 25 57 PM](https://user-images.githubusercontent.com/47183489/71842805-e5653980-3098-11ea-9af1-f87d5062eb9d.png)